plugins/copr.py

```diff
@@ -148,6 +148,8 @@ def run(self):
 
         repo_filename = "{}/_copr_{}-{}.repo" \
                         .format(self.base.conf.get_reposdir, copr_username, copr_projectname)
+        modules_repo_filename = "/etc/fm.modules.d/_copr_{}-{}.cfg" \
```

Comment:
the main problem here is that this directory doesn't exist on most of machines... either you should add some Requires or own directory or ...